### ISSUE TITLE
Hope to move to UWP Modern .NET in the future

### ISSUE THREAD

**Issue by Gaoyifei1011** [dongle-the-gadget/WinverUWP#53]
Hope to move to UWP Modern .NET in the future
希望未来能迁移到 UWP Modern .NET

Microsoft 已经官宣了 UWP Modern .NET 的支持，希望未来能将 WinverUWP 迁移到 Modern .NET
Mikrosoft has officially announced the support of UWP Modern .NET and hopes to migrate to UWP Modern .NET in the future

https://devblogs.microsoft.com/ifdef-windows/preview-uwp-support-for-dotnet-9-native-aot/

**Comment by dongle-the-gadget** [comment]
I do know about this, however the lack of WinUI 2 (currently) is a blocker.

**Comment by Gaoyifei1011** [comment]
> I do know about this, however the lack of WinUI 2 (currently) is a blocker.

Indeed, the lack of winui2 is currently a major barrier to upgrading to modern .NET. If you really want to experiment with winui2 modern .net, you can use the driver1998 DisposableMemory.ModernNetUAP.WinUI nuget package to test it out until Microsoft officially supports it
确实，目前缺少 winui2 的确是升级到 modern .net 的一大障碍。如果您真的想要使用 winui2 modern .net 尝鲜，您可以使用 driver1998的 DisposableMemory.ModernNetUAP.WinUI nuget包来进行测试，直到微软正式支持

link
https://www.nuget.org/packages/DisposableMemory.ModernNetUAP.WinUI
https://www.nuget.org/packages/DisposableMemory.ModernNetUAP.XamlCompiler

**Comment by dongle-the-gadget** [comment]
I haven’t managed to get that package working.

**Comment by Gaoyifei1011** [comment]
> I haven’t managed to get that package working.

You can see driver1998's sample:https://github.com/driver1998/NetCoreApp

**Comment by driver1998** [comment]
> I haven’t managed to get that package working.

My WinUI 2 projection package did not work with WASDK single project MSIX packaging, since it provides its own WebView2 projection, which conflicts with the one implicitly referenced in WASDK.

The new preview should fix this (it now uses projections from a preview version of WebView2 nuget).
https://www.nuget.org/packages/DisposableMemory.ModernNetUAP.WinUI/2.8.6.3-preview
